src/inmanta/ast/constraint/expression.py

```diff
@@ -320,28 +322,28 @@ def _un_op(self, arg):
         return not arg
 
 
-class Regex(BinaryOperator):
+class Regex(UnaryOperator):
```

Comment:
Could you add the `stableapi` decorator and add a test to `tests/test_import_entry_points.py`? I think this change is safe but I just discovered that this class is used by `lsm/plugins/typedef.py`. We should make sure we stay aware of this.

Reply:
done!

Reply:
It looks like you forgot about the test case.